packages/menu/src/MenuItem.tsx

```diff
@@ -192,6 +191,7 @@ const MenuItem = React.forwardRef(
           {...anchorProps}
           {...rest}
           className={cx(
+            menuItemContainerClassName,
```

Comment:
why are we changing where this is defined?

Reply:
This was initially on this element but was moved to the glyph in this [PR](https://github.com/mongodb/leafygreen-ui/commit/f2d63a60ed0d8669f44fd8e6f45af827b875414f#diff-3f4788e0b950199ad21de29f52ccc19e90bf4acf14da596954b54da22dc966f6L134). Also, sased on the name, it's suppose to be on the menu item container. I think we need to spend some time cleaning up this component.

Reply:
👍